doc/user-docs/UsingCommandlineArguments.md

```diff
@@ -125,4 +126,67 @@ This creates a new Windows Terminal window with one tab, and 3 panes:
   and will use `wsl.exe` as the commandline (instead of the default profile's
   `commandline`).
 
+
+### Using multiple commands from PowerShell
+
+The Windows Terminal uses the semicolon character `;` as a delimiter for
+separating subcommands in the `wt` commandline. Unfortunately, `powershell` also
+uses `;` as a command separator. To work around this you can use the following
+tricks to help run multiple wt sub commands from powershell. In all the
+following examples, we'll be creating a new Terminal window with three panes -
+one running `cmd`, one with `powershell`, and a last one running `wsl`.
+
+#### Single quoted parameters (if you aren't calculating anything):
+In this example, we'll wrap all the parameters to `wt` in single quotes (`'`),
+and run `wt` with the `Start-Process` command:
+
+```PowerShell
+start wt 'new-tab "cmd"; split-pane -p "Windows PowerShell" ; split-pane -H wsl.exe'
+```
+
+#### Escaped quotes (if you need variables):
+
+If you'd like to pass a value contained in a variable to the `wt` commandline,
+instead use the following syntax:
+
+```PowerShell
+$ThirdPane = "wsl.exe"
+start wt "new-tab cmd; split-pane -p `"Windows PowerShell`" ; split-pane -H $ThirdPane"
+```
+
+Note the usage of  `` ` `` to escape the double-quotes (`"`) around "Windows
+Powershell" in the `-p` parameter to the `split-pane` sub-command.
+
+#### Not using `start`
+
+The astute reader will note that all the above examples used `start` to launch the Terminal.

```

Comment:
eh. We could probably do without this statement.

Reply:
As a counterpoint? I saw `start` and wondered "wtf is he using start for over and over"

Reply:
Ah. By "statement" I mean this specific sentence, not section. Feels a bit weird to call the reader "astute" haha.

Reply:
no, i mean, i _specifically_ noticed that `start` was in use, and I wanted to know why. I think this _sentence_ is valuable because it makes people feel seen ;P

Reply:
Um.... It's probably worth noting that `start` avoids the **waiting** problem described below.